bigtable/cmd/cbt/testing.go

```diff
@@ -48,3 +54,28 @@ func captureStdout(f func()) string {
 	w.Close()
 	return <-outC
 }
+
+func assertEqual(t *testing.T, got, want interface{}, opts ...cmp.Option) {
```

Comment:
nit: Please avoid adding asset functions and validate the function inline. There are a couple different options: just use the stdlib and t.Error, make use of testutil, or cmp.Diff/Equal may be worth using.

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures